src/widgets/modals/SearchForm.js

```diff
@@ -111,7 +112,10 @@ const mapStateToProps = state => {
 
 const mapDispatchToProps = dispatch => ({
   // TODO: update to use PatientActions.updatePatient()
```

Comment:
This `TODO` and the lower one, for prescribers, appear to be done. If so, can you delete them please.

Reply:
Done! 13548bd.